### DISCUSSION TITLE
Creating a directory named "doc" to store ".md" files does not allow them to be loaded onto the webpage.   在 "doc"目录下面创建一个用于存放 “.md” 的文件无法，无法加载到网页上

### DISCUSSION THREAD

**Discussion by shuihuahaiyang** [vuepress/core#1618] [Q&A]
![question](https://github.com/user-attachments/assets/f2490ce2-4e1a-4ed5-bd74-9607bde9b330)

The Markdown files in the not1 folder are not loading. According to the official documentation, Markdown files should be loadable from any subfolder within the doc directory.
Official screenshot
这个not1文件夹的md文件无法加载，我看官网是doc文件下面任意文件夹都可以加载。
官方的截图  

![image](https://github.com/user-attachments/assets/24c74817-decf-4101-bb8e-50d64a4d5c4e)

**Comment by Mister-Hope** [comment]
Need reproduction

**Comment by Mister-Hope** [comment]
Closing as no response
